Sources/Client/Client/Client.swift

```diff
@@ -255,6 +255,15 @@ public final class Client: Uploader {
         
         if appState == .active {
             webSocket.connect()
+            
+            // The improve latency, call the preheat endpoint to set up TCP connection which can be reused.
+            // We delay the call a little bit to make sure the WS connection is fired before this one.
+            DispatchQueue.global().async {
```

Comment:
I think you can safely use `webSocket.provider.callbackQueue`

Reply:
There's no reason for this really. The `engine.connect` is called on the main queue. I just want to be sure this call happens a little bit later.